packages/amplify-codegen/src/commands/models.js

```diff
@@ -145,6 +146,13 @@ function getModelOutputPath(context) {
   }
 }
 
+function getAmplifyCLIVersionComment(context) {
+  if (context.usageData == null || !context.usageData.version) {
+    return null
+  }
+  return 'Generated using amplify-cli-version: ' + context.usageData.version;
```

Comment:
CLI uses caret version for amplify-codegen. I think it is better to include this information as well in the comment, which is `Generated using amplify-cli Vxx.xx.xx with amplify-codegen Vxx.xx.xx`

Reply:
Added the codegen version. I could only find the information from the context using [this logic](https://github.com/aws-amplify/amplify-codegen/pull/116/files#diff-9cdc392599e83dc816cb33512961b1b923e38faa0692da825ccdaa048081cc6bR159).